frontend/src/components/Issue/IssueActivityPanel.vue

```diff
@@ -479,6 +484,10 @@ const doCreateComment = (comment: string, clear = true) => {
     if (!isSubscribed) {
       addSubscriberId(currentUser.value.id);
     }
+
+    if (comment === "LGTM") {
```

Comment:
Do we need to check for user role or lgtm configs of related project? @RainbowDashy

Reply:
Agreed. But it might be a repeated implementation if we check it at the front-end. So I prefer we run the check whatever the current user's role is. And it will be better if we can run just the LGTM check and keep the other types of checks as-is.

Reply:
Let's add it later if needed.